src/react/src/util/constants.jsx

```diff
@@ -1272,18 +1272,3 @@ export const facilityClaimStepsNames = Object.freeze({
     SUPPORT_DOC: 'Support Documentation',
     ADDITIONAL_DATA: 'Additional Data',
 });
-
-export const componentsWithErrorMessage = Object.freeze({
```

Comment:
Why did you decide to delete `componentsWithErrorMessage` ?

Reply:
I added this componentsWithErrorMessage in the previous PR. But the rest of errors messages(from BE) stayed as string and that became the reason of the bug.

Reply:
Then, you can make each error message just a string.

Reply:
Then you can make each error message just a string.